stable/fairwinds-insights/README.md

```diff
@@ -0,0 +1,88 @@
+| dashboard.nodeSelector | object | `{}` | Node Selector for the front end pods. |
+| dashboard.tolerations | list | `[]` | Tolerations for the front end pods. |
+| api.port | int | `8080` | Port for the API server to listen on. |
+| api.pdb.enabled | bool | `false` | Create a pod disruption budget for the API server. |
+| api.hpa.enabled | bool | `false` | Create a horizontal pod autoscaler for the API server. |
+| api.hpa.min | int | `2` | Minimum number of replicas for the API server. |
+| api.hpa.max | int | `4` | Maximum number of replicas for the API server. |
+| api.hpa.cpuTarget | int | `50` | Target CPU utilization for the API server. |
+| api.resources | object | `{"limits":{"cpu":"1000m","memory":"1024Mi"},"requests":{"cpu":"250m","memory":"256Mi"}}` | Resources for the API server. |
+| api.nodeSelector | object | `{}` | Node Selector for the API server. |
+| api.tolerations | list | `[]` | Tolerations for the API server. |
+| dbMigration.resources | object | `{"limits":{"cpu":1,"memory":"1024Mi"},"requests":{"cpu":"80m","memory":"128Mi"}}` | Resources for the database migration job. |
+| alertsCronjob.resources | object | `{"limits":{"cpu":"500m","memory":"1024Mi"},"requests":{"cpu":"80m","memory":"128Mi"}}` | Resources for the Slack/Datadog integrations |
+| alertsCronjob.schedules | list | `[{"cron":"5/10 * * * *","interval":"10m","name":"realtime"},{"cron":"0 16 * * *","interval":"24h","name":"digest"}]` | CRON schedules for the Slack/Datadog integrations |
+| aggregateCronjob.resources | object | `{"limits":{"cpu":"250m","memory":"512Mi"},"requests":{"cpu":"40m","memory":"32Mi"}}` | Resources for the Workload Metrics aggregation job. |
+| aggregateCronjob.schedules | list | `[{"cron":"5 0/2 * * *","interval":"120m","name":"bi-hourly"}]` | CRON schedules for the Workload Metrics aggregation job. |
+| emailCronjob.resources | object | `{"limits":{"cpu":"500m","memory":"1024Mi"},"requests":{"cpu":"80m","memory":"128Mi"}}` | Resources for the Action Items email job. |
+| emailCronjob.schedules | list | `[{"cron":"0 16 * * 1","interval":"168h","name":"weekly-email"}]` | CRON schedules for the Action Items email job. |
+| deleteOldActionItemsCronjob.resources | object | `{"limits":{"cpu":"500m","memory":"1024Mi"},"requests":{"cpu":"80m","memory":"128Mi"}}` | Resources for the delete old Action Items job. |
+| deleteOldActionItemsCronjob.schedules | list | `[{"cron":"0 0 * * *","interval":"24h","name":"ai-cleanup"}]` | CRON schedules for the delete old Action Items job. |
+| service.port | int | `80` | Port to be used for the API and Dashboard services. |
+| sanitizedBranch | string | `nil` | Prefix to use on hostname. Generally not needed. |
+| ingressApi.annotations | list | `[]` | Annotations to add to the API ingress. |
+| ingress.enabled | bool | `false` | Install Ingress objects. |
```

Comment:
I wonder if we might want to consider separating these? Someone might want an ingress for the dashboard, but not the api for example...